srcpkgs/onedrive/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'onedrive'
+pkgname=onedrive
+version=2.4.3
+revision=1
+wrksrc="onedrive-${version}"
+build_style=gnu-configure
+configure_args="DC=ldc2 --bindir=/usr/bin"
+depends="curl"
```

Comment:
Move `depends` after `hostmakedepends` and `depends`.

Reply:
removing `depends` at all as it's invalid: https://github.com/void-linux/void-packages/pull/23898#discussion_r461682294